lib/form1010_ezr/service.rb

```diff
@@ -136,14 +138,32 @@ def post_fill_veteran_full_name(parsed_form)
 
       StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.missing_full_name")
 
-      parsed_form['veteranFullName'] = @user.full_name_normalized&.stringify_keys
+      parsed_form['veteranFullName'] = @user.full_name_normalized&.compact&.stringify_keys
       parsed_form
     end
 
-    def post_fill_fields(parsed_form)
-      post_fill_required_fields(parsed_form)
+    # Due to issues with receiving submissions that do not include the Veteran's SSN, we'll
+    # try to add it in before we validate the form
+    def post_fill_veteran_ssn(parsed_form)
+      return if parsed_form['veteranSocialSecurityNumber'].present?
+
+      StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.missing_ssn")
+
+      parsed_form['veteranSocialSecurityNumber'] = @user.ssn_normalized
+      parsed_form
```

Comment:
Maybe I'm misreading it, but why are we returning the parsed_form object here? It looks like we aren't actually doing anything with any returned value anywhere we call this method, but are instead just mutating the passed in parsed_form object?